base/nullable.jl

```diff
@@ -45,16 +45,22 @@ function show{T}(io::IO, x::Nullable{T})
     end
 end
 
-get(x::Nullable) = x.isnull ? throw(NullException()) : x.value
-
-@inline function get{T}(x::Nullable{T}, y)
-    if isbits(T)
-        ifelse(x.isnull, convert(T, y), x.value)
+"""
+    get(x::Nullable[, y])
+
+Attempt to access the value of `x`. Returns the value if it is present;
+otherwise, returns `y` if provided, or throws a `NullException` if not.
+"""
+@inline function get{S,T}(x::Nullable{S}, y::T)
+    if isbits(S)
+        ifelse(x.isnull, y, x.value)
```

Comment:
Not sure. For `isbits` types, it should always be safe to eagerly evaluate `x.value`, even if it's nonsense. Maybe we don't need `null_safe_op` here (what's "op" BTW?).